democracy/migrations/0016_merge_comments.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-05-18 13:18
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def backwards(apps, schema_editor):
+    pass
+
+
+def forwards(apps, schema_editor):
+    """
+    Convert Hearing comments to introduction Section comments
+    """
+    hearing_comments = apps.get_model('democracy', 'HearingComment').objects.all()
+    section_comment_model = apps.get_model('democracy', 'SectionComment')
+
+    for hearing_comment in hearing_comments:
+        data = {field: getattr(hearing_comment, field) for field in ('content', 'author_name', 'n_votes', 'created_by',
+                                                                     'created_at', 'modified_at', 'modified_by')}
+        section = hearing_comment.hearing.sections.get(type__identifier='introduction')
+        data['section'] = section
+
+        section_comment_model.objects.create(**data)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('democracy', '0015_add_organization_model'),
+    ]
+
+    operations = [
+        migrations.RunPython(forwards, backwards),
```

Comment:
Use `migrations.RunPython.noop` as backwards (saves you from writing the same fn yourself :) )